### ISSUE TITLE
Error trying to run Logitech-CSGO

### ISSUE THREAD

**Issue by jootrnt** [antonpup/Logitech-CSGO#11]
I did exactly the way was in the tutorial here. Running as administrator doesn't work.
I'm using a G910 with Windows 10 x64 and Logitech Gaming Software 8.79.77.

Message:
There was an error initializing Logitech LED SDK
"Unable to load DLL 'LogitechLedEnginesWrapper' : The specified module could not be found (Exception from HRESULT: 0x8007007E"

**Comment by antonpup** [comment]
Yes, this should be addressed in the next version. I will release it sometime next week.

**Comment by jootrnt** [comment]
My friend, thanks for the reply, I fixed this searching around on a logitech forum, someone got this error with another plugin and he fixed this installing Visual C++ Redistributable for Visual Studio 2012 Update 4 (https://www.microsoft.com/en-us/download/details.aspx?id=30679). I already had the 2013 version installed but the program wasn't running, now with the 2012 its fine. Its very nice this program you created, keep the good work, I only bought this keyboard because I saw a video of your program.
